modules/op-tkn-hub-interaction.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// *  cli_reference/tkn_cli/op-tkn-reference.adoc
+
+[id="op-tkn-hub-interaction_{context}"]
+= Hub interaction commands
+
+Interact with Tekton Hub for resources such as tasks and pipelines.
+
+== hub
+Interact with hub.
+
+.Example: Display help
+[source,terminal]
+----
+$ tkn hub -h
+----
+
+.Example: Interact with hub API server
+[source,terminal]
+----
+$ tkn hub --api-server https://api.hub.tekton.dev
+----
+
+[NOTE]
+====
+For each command examples given below, you can get the corresponding sub-commands and flags by executing `tkn hub <command> --help`.
+====
+
+== hub downgrade
+Downgrade an installed resource
+
+.Example: Downgrade an installed resource
```

Comment:
Downgrade an installed task `mytask` to  version `0.2`

Reply:
Downgrade an installed task `mytask` to version `newversion`

Reply:
@VeereshAradhya I thought instead of keeping a placeholder giving an example with a value would be helpful to understand.
but if placeholder is fine. I don't have any strong opinion against it, fine with me :)
same for rest of the comments.